src/test/java/io/zipcoder/interfaces/ZipCodeWilmingtonTest.java

```diff
@@ -0,0 +1,50 @@
+package io.zipcoder.interfaces;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ZipCodeWilmingtonTest {
+	
```

Comment:
Consider this structure instead...
```java
//Given
String instructorName = "Brendan";
Instructor expected = new Instructor(instructorName)

//When
ZipCodeWilmington.hire(expected);

//Then
Instructor actual = (Instructor) Classroom.get(instructorName);
Assert.assertEquals(expected, actual);
Instructor instructor = ClassRoom.getRoster().get(expected);
```
Contemplate why this will not work with our `ClassRoom`'s current Object orientation